docs/extend/modular-resources/create/_index.md

```diff
@@ -16,55 +16,60 @@ description: "Use the Viam module system to implement modular resources that can
+   This will help you ensure that the client interface you create returns the expected response for viam-server.
 
-    **To prepare to code a new resource model**:
+   - Find the relevant `viam-server` client interface as `<resource-name>/client.go` or `<resource-name>/client.py` on [Viam's GitHub](https://github.com/viamrobotics/rdk/blob/main/).
+     - For example, the base client is defined in [<file>rdk/components/base/client.go</file>](https://github.com/viamrobotics/rdk/blob/main/components/base/client.go).
+       See [Valid APIs to implement in your model](/extend/modular-resources/key-concepts/#valid-apis-to-implement-in-your-model) for more information.
+   - Use this first file as a reference when making edits to `my_modular_resource.go` or `my_modular_resource.py`.
+   - Name your model according to the namespace of the built-in API you are implementing.
+     For more information, see [Naming your model](/extend/modular-resources/key-concepts/#naming-your-model).
 
-    The methods you will code in <file>my_modular_resource.go</file> or <file>my_modular_resource.py</file> are your model's "**client** interface", or how your model's server will respond when `viam-server` asks your resource for something through the API.
+1. **Prepare to Import Your Custom Model and Subtype's API:**
 
-    View the appropriate `viam-server` client interface to see what your resource's responses from `viam-server` will look like when your model is utilizing the subtype's API.
-    This way, you can make the client interface you code return the type of response `viam-server` expects to receive.
+   To prepare to import your custom model and chosen resource subtype's API into your main program and register them with your preferred SDK:
 
-    - Find the relevant `viam-server` client interface as `<resource-name>/client.go` or `<resource-name>/client.py` on [Viam's GitHub](https://github.com/viamrobotics/rdk/blob/main/).
-      See [Valid APIs to implement in your model](/extend/modular-resources/key-concepts/#valid-apis-to-implement-in-your-model) for more information.
-    - For example, the base client is defined in [<file>rdk/components/base/client.go</file>](https://github.com/viamrobotics/rdk/blob/main/components/base/client.go).
-    - Base your edits to <file>my_modular_resource.go</file> or <file>my_modular_resource.py</file> on this first file.
-    - Name your model according to the namespace of the built-in API you are implementing.
-      For more information see [Naming your model](/extend/modular-resources/key-concepts/#naming-your-model).
+   - Find the subtype API as defined in the relevant `<resource-name>/<resource-name>.go` file in the RDK on Viam's GitHub.
+     - For example, the base subtype is defined in [<file>rdk/components/base/base.go</file>](https://github.com/viamrobotics/rdk/blob/fdff22e90b8976061c318b2d1ca3b1034edc19c9/components/base/base.go#L37).
```

Comment:
This may mean the line number could end up being wrong - but it will ensure the file is not out of date
```suggestion
     - For example, the base subtype is defined in [<file>rdk/components/base/base.go</file>](https://github.com/viamrobotics/rdk/blob/main/components/base/base.go#L37).
```